todo-app/src/TodoItem.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default class TodoItem extends React.Component {
+  render() {
+    return (
+      <div><ul>
```

Comment:
If each `TodoItem` renders a `ul` then for three items we will have three `ul` elements, which is not very nice, right? Ideally we want to have one `ul` and each item should render an `li` element?